generated/datafactory/microsoft.datafactory/2018-06-01/types.md

```diff
@@ -2798,6 +2978,7 @@
 ### TumblingWindowTriggerDependencyReference
 #### Properties
 * **offset**: string: Timespan applied to the start time of a tumbling window when evaluating dependency.
+* **referenceTrigger**: [TriggerReference](#triggerreference) (Required): Referenced trigger.
```

Comment:
`TumblingWindowTriggerDependencyReference` is the grandchild of `DependencyReference`, and this property is defined on the branch's parent, [`TriggerDependencyReference`](https://github.com/Azure/azure-rest-api-specs/blob/37c9b56303da9ec425b00575c0027b44d11a772f/specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Trigger.json#L543).